iot_config/src/main.rs

```diff
@@ -136,11 +130,7 @@ impl Daemon {
             .serve_with_shutdown(listen_addr, shutdown_listener)
             .map_err(Error::from);
 
-        tokio::try_join!(
-            db_join_handle.map_err(Error::from),
-            md_pool_handle.map_err(Error::from),
-            server
-        )?;
+        tokio::try_join!(server)?;
```

Comment:
nothing to join here anymore; i think you can just await this

Reply:
fixed here 683b9d51710e605db82d0a2dc358c69d4380c5c3